tests/testsuite/env.rs

```diff
@@ -790,12 +789,15 @@ mod unicode_tests {
     }
 
     #[test]
-    #[should_panic(expected = r#"`Result::unwrap()` on an `Err` value: "\xFF""#)]
     fn test_invalid_unicode_value_not_filtered() {
         temp_env::with_vars(
             vec![("invalid_value1", Some(make_invalid_unicode_os_string()))],
             || {
-                Environment::default().collect().unwrap();
+                let result = Environment::default().collect();
+
+                assert!(
+                    matches!(result, Err(ConfigError::Message(msg)) if msg == r#"key "invalid_value1" is not valid Unicode: "\xFF""#)
+                );
```

Comment:
Could you instead assert this using a snapshot test, like https://github.com/rust-cli/config-rs/blob/370c939a1ad6d5b52dcfaa413604ba921773515c/tests/testsuite/errors.rs#L46-L49

Reply:
Done. Also fixed the windows test, I think.